### PR TITLE
Collijk/feature/new ode postprocessing

### PR DESCRIPTION
**Pull request by collijk**

### CODE REVIEW

File: src/covid_model_seiir_pipeline/lib/io/marshall.py [hunk 1]
```diff
@@ -76,7 +77,10 @@ def dump(cls, data: pd.DataFrame, key: DatasetKey, strict: bool = True) -> None:
     @classmethod
     def load(cls, key: DatasetKey) -> pd.DataFrame:
         path, columns = cls._resolve_key(key)
-        data = pd.read_parquet(path, columns=columns, engine='fastparquet')
+        with warnings.catch_warnings():
```
Comment (kathrynline) [inline src/covid_model_seiir_pipeline/lib/io/marshall.py:80]: OOh I've never seen this package before.

File: src/covid_model_seiir_pipeline/pipeline/diagnostics/model/plotter.py [hunk 2]
```diff
@@ -173,6 +191,117 @@ def make_covariates_page(plot_versions: List[PlotVersion],
     write_or_show(fig, plot_file)
 
 
+def make_variants_page(plot_versions: List[PlotVersion],
```
Comment (kathrynline) [inline src/covid_model_seiir_pipeline/pipeline/diagnostics/model/plotter.py:194]: returns None?

File: src/covid_model_seiir_pipeline/pipeline/forecasting/model/ode_forecast.py [hunk 3]
```diff
@@ -89,7 +90,31 @@ def build_model_parameters(indices: Indices,
     )
 
 
-def adjust_vaccinations_for_variants(vaccine_data: pd.DataFrame, covariates: pd.DataFrame):
+def adjust_vaccinations_for_variants(vaccine_data: pd.DataFrame, covariates: pd.DataFrame, system: str):
```
Comment (kathrynline) [inline src/covid_model_seiir_pipeline/pipeline/forecasting/model/ode_forecast.py:93]: Returns dict?

File: src/covid_model_seiir_pipeline/pipeline/forecasting/model/ode_forecast.py [hunk 4]
```diff
@@ -89,7 +90,31 @@ def build_model_parameters(indices: Indices,
...
+    if 'variant' in system:
+        return _adjust_variant_vaccines(vaccine_data)
+    else:
+        return _adjust_non_variant_vaccines(vaccine_data, covariates)
+
+
+def _adjust_variant_vaccines(vaccine_data):
```
Comment (kathrynline) [inline src/covid_model_seiir_pipeline/pipeline/forecasting/model/ode_forecast.py:100]: Returns dict?

**Review by kathrynline — APPROVED** [review]

**Comment by collijk** [comment]
Not ready for review

**Pull request closed without merging**